src/EndpointStandalone.sol

```diff
@@ -28,11 +29,22 @@ abstract contract EndpointStandalone is
         _;
     }
 
+    /// @dev Returns the owner of the manager contract
+    function getManagerOwner() public view returns (address) {
+        return IManager(manager).getOwner();
+    }
+
+    /// @dev transfer the ownership of the endpoint to a new address
+    function transferEndpointOwnership(address newOwner) external {
```

Comment:
This needs to change to have an `onlyManager` modifier guard and the call should go to `_transferOwnership` internal method. Currently this will always revert on calls from the manager because the manager is not the owner. 

We want the owner of the endpoints to stay in sync with the manager owner so we also need to override `transferOwnership` and `renounceOwnership` to do nothing (to prevent the owner from going out of sync with the manager owner).

We'll also need to override `renounceOwnership` (or get rid of it altogether) in the Manager to prevent the the manager owner from being renounced as I don't think we should support that case, what do you think?

Reply:
overrode the methods directly in the concrete implementations